examples/server/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020 DigiByte Foundation NZ Limited
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+const RosettaSDK = require('../..');
+
+// Create a server configuration
+const Server = new RosettaSDK.Server({
+  URL_PORT: 8080,
+});
+
+Server.launch()
+  .catch(e => logger.error(e));
```

Comment:
@SmartArray ,

Can you add a new line here?  I'll take a look at the project structure but we should be using ES Lint with Semistandard rules.  Most IDEs would catch this as a result and offer to automatically fix them.

Reply:
Correct.